core-bundle/src/Image/PictureFactory.php

```diff
@@ -157,9 +157,12 @@ private function createConfig(array|int|string|null $size): array
                 if (null !== $imageSizes) {
                     $options->setSkipIfDimensionsMatch((bool) $imageSizes->skipIfDimensionsMatch);
 
-                    if (!$imageSizes->preserveMetadata) {
+                    if ('metaDelete' === $imageSizes->preserveMetadata) {
                         $options->setPreserveCopyrightMetadata([]);
-                    } elseif ($preserveMetadata = StringUtil::deserialize($imageSizes->metadata, true)) {
+                    } elseif (
+                        'metaOverwrite' === $imageSizes->preserveMetadata
+                        && ($preserveMetadata = StringUtil::deserialize($imageSizes->metadata, true))
```

Comment:
Maybe we can just switch they keys?

```suggestion
                        'metaOverwrite' === $imageSizes->metadata
                        && ($preserveMetadata = StringUtil::deserialize($imageSizes->preserveMetadata, true))
```

Reply:
I think I’d like `preserveMetadata` and `metadataToPreserve`
or `preserveMetadata` and `metadataKeys`
better, WDYT?

Reply:
I like `metadataKeys`, however, I would also like the `preserveMetadata` key to have the same meaning in both the predefined image size and `tl_image_size`.